indexing-hadoop/src/test/java/io/druid/indexer/BatchDeltaIngestionTest.java

```diff
@@ -340,13 +340,16 @@ private HadoopDruidIndexerConfig makeHadoopDruidIndexerConfig(Map<String, Object
         new HadoopIngestionSpec(
             new DataSchema(
                 "website",
-                new StringInputRowParser(
-                    new CSVParseSpec(
-                        new TimestampSpec("timestamp", "yyyyMMddHH", null),
-                        new DimensionsSpec(ImmutableList.of("host"), null, null),
-                        null,
-                        ImmutableList.of("timestamp", "host", "host2", "visited_num")
-                    )
+                MAPPER.convertValue(
```

Comment:
Does this work recursively? Or would you get a Map where the values are regular objects rather than other Maps?